easybuild/tools/github.py

```diff
@@ -395,6 +407,235 @@ def post_comment_in_issue(issue, txt, repo=GITHUB_EASYCONFIGS_REPO, github_user=
         raise EasyBuildError("Failed to create comment in PR %s#%d; status %s, data: %s", repo, issue, status, data)
 
 
+def _copy_easyconfigs_to_repo(paths, target_dir):
```

Comment:
However, let me move this function (and make it non-private), it doesn't really belong in `github.py`.
I'll move it to `easyconfig.py`, where related functions like `robot_find_easyconfig` are already located.